src/main/resources/config/liquibase/changelog/20200701600000_add_sorting_in_pivot_table.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet author="khushbu" id="200701600000">
+
+        <update tableName="property_types">
+            <column name="color_picker_default_value" value='787878' />
+            <where>id = 677</where>
```

Comment:
can it happen that this id is not 677 for the customer? Should we use some other property for comparison?

Reply:
no, 677 is the id of the property